modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -241,6 +241,17 @@
 		end
 	end
 
+--
+-- Write out the additional props.
+--
+
+	function dotnetbase.additionalProps(cfg)
+		for i = 1, #cfg.props do
+			for key, value in spairs(cfg.props[i]) do
+				_p(2, '<%s>%s</%s>', key, value, key)
```

Comment:
Wonder if we have a xml escaper for `value`...
It seems not according to above function (which is more likely to have `<` or `>`.)

Reply:
There are multiple xml escaping functions, which one should I use?
For now, I use `vs2005.esc` for csharp, `vs2010.esc` for c++